### PR TITLE
Added example to conditional aggregation docs.

### PR DESCRIPTION
**Pull request by vidyarani-dg**

### CODE REVIEW

File: docs/ref/models/conditional-expressions.txt [hunk 1]
```diff
@@ -236,3 +236,21 @@ On other databases, this is emulated using a ``CASE`` statement:
...
+
+Suppose we want to get the total discount for all clients based on the
+``account_type``. We can use the conditional expression in the
+:ref:`aggregate functions <aggregation-functions>` to achieve this::
+
+    >>> a_month_ago = date.today() - timedelta(days=30)
+    >>> a_year_ago = date.today() - timedelta(days=365)
+    >>> # Get the discount for each Client based on the account type
+    >>> Client.objects.aggregate(
+    ...     total_discount=Coalesce(Sum(Case(
```
[the remark this answers is not in the document]
Comment (vidyarani-dg) [reply docs/ref/models/conditional-expressions.txt]: Will do, thanks.

**Comment by timgraham** [comment]
If I understand correctly, the purpose is to re-add an example similar to what was there before b78d100fa62cd4fbbc70f2bae77c192cb36c1ccd -- is that correct? I'm not sure the example is useful -- looking at the other examples, it seems a platinum account gets a 10% discount, gold 5%, etc. In that case, the result of 21% isn't really meaningful in anyway, I don't think?

**Comment by vidyarani-dg** [comment]
@timgraham : I apologize for my tardiness in getting back to this comment, I have been on holiday for a while. 

Yes, the purpose of this documentation is to re-add an example that was there already in the previous version of the docs. I was looking for something similar and I couldn't find it in the docs. I was told that a similar example was there in the previous versions. Hence I thought it would be useful for others if I added this. 

The conditional aggregation docs (https://docs.djangoproject.com/en/2.0/ref/models/conditional-expressions/#conditional-aggregation)  already creates 3 instances - 2 of which are Platinum and one is Regular, hence the discount is 21%. Does this make sense ?

**Comment by timgraham** [comment]
What I meant is that I don't think the "sum of the percent discount for all clients" is a realistic use case. That number doesn't have any usefulness.

**Comment by timgraham** [comment]
Closing due to inactivity.

**Pull request closed without merging**